### DISCUSSION TITLE
Can I make Regolith use the gnome input source for keybindings?

### DISCUSSION THREAD

**Discussion by gerbigtim** [regolith-linux/regolith-desktop#846] [Q&A]
Hello,

I encountered a problem when using regolith with a different keyboad layout. The keyboard shortcuts don't seem to update when switching to a different keyboard layout using <kbd>Super</kbd>+<kbd>Alt</kbd>+<kbd>Backspace</kbd> (`us` to `us+colemak` in my case).
Is there a way to use the gnome input source as the base for regolith keyboard shortcuts?

Tanks in advance
Tim

**Comment by gerbigtim** [comment]
It seems like regolith uses the 1st input source listed in Settings > Keyboard for its keybindings, no matter what the current input source is. 
So I can adjust the keybindings by switching the order in the settings.

This is a workaround but it doesn't feel like its the intended behavior, since programs usually use the current input method for their keybindings.

**Reply by kgilmer** [reply]
Hi @gerbigtim , I don't use multiple layouts myself so am not familiar with this.  When you say "use the current input method" are you implying some other UI than `settings`  that's used to switch the keyboard layout?  Or that all layouts should be concurrently "active"?

**Reply by gerbigtim** [reply]
Hi @kgilmer, thank you for your reply!
I only switch the active keyboard layout by using <kbd>Super</kbd>+<kbd>Alt</kbd>+<kbd>Backspace</kbd>. This cycles through the list of keyboard layouts found in `Settings` > `Keyboard` > `Input Sources`. Only one layout is active at the same time. 
Regolith keybindings use the first layout in `Settings` > `Keyboard` > `Input Sources`, which is not necessarily the same as the currently active keyboard layout.

**Reply by tartopohm** [reply]
I am experimenting the same issue, and that workaround works for me as well, changing the layouts order in `Settings` > `Language and Region` > `Input Sources`.
